README.md

```diff
@@ -59,135 +60,66 @@ with sycl_context("gpu"):
+||**LinearRegression**|All parameters except `normalize` != False and `sample_weight` != None. | Only dense data is supported, `#observations` should be >= `#features`. |
+||**Ridge**|All parameters except `normalize` != False, `solver` != 'auto' and `sample_weight` != None. | Only dense data is supported, `#observations` should be >= `#features`. |
+||**ElasticNet**|All parameters are supported except `sample_weight` | Multi-output and sparse data is not supported, `#observations` should be >= `#features`. |
+||**Lasso**|All parameters are supported except `sample_weight`| Multi-output and sparse data is not supported, `#observations` should be >= `#features`. |
+|Clustering|**KMeans**|All parameters except `precompute_distances` and `sample_weight` != None. | No limitations. |
+||**DBSCAN**|Supported `metric` = 'euclidean' and 'minkowski' with p='2', `algorithm`='brute. Other parameters are fully supported. | Only dense data is supported. |
+|Dimensionality reduction|**PCA**|All parameters except `svd_solver` != 'full'. | No limitations. |
+|Other|**train_test_split**|All parameters are supported. | Only dense data is supported.|
+||**assert_all_finite**|All parameters are supported. | Only dense data is supported. |
+||**pairwise_distance**|With `metric`='cosine' and 'correlation'.| Only dense data is supported. |
+
+Scenarios that are only available in the `master` branch (not released yet):
+
+|Task|Functionality|Parameters support|Data support|
+|:---|:------------|:-----------------|:-----------|
+|Regression|**KNeighborsRegressor**|Supported `metric` = 'euclidean' and 'minkowski' with `p` = 2. Other parameters are fully supported. | Sparse data is not supported. |
+|Unsupervised|**NearestNeighbors**|Supported `metric` = 'euclidean' and 'minkowski' with `p` = 2. Other parameters are fully supported. | Sparse data is not supported. |
+|Dimensionality reduction|**TSNE**|Supported `metric` = 'euclidean' and 'minkowski' with `p` = 2. Other parameters are fully supported. | Sparse data is not supported. |
+|Other|**roc_auc_score**|Parameters `average`, `sample_weight`, `max_fpr` and `multi_class` are not supported. | No limitations. |
+
+## scikit-learn verbose
+
+To find out which implementation of the algorithm is currently used (daal4py or stock Scikit-learn), set the environment variable:
+- On Linux and Mac OS: `export IDP_SKLEARN_VERBOSE=INFO`
+- On Windows: `set IDP_SKLEARN_VERBOSE=INFO`
+
+For example, for DBSCAN you get one of these print statements depending on which implementation is used:
+- `INFO: sklearn.cluster.DBSCAN.fit: uses Intel(R) oneAPI Data Analytics Library solver`
+- `INFO: sklearn.cluster.DBSCAN.fit: uses original Scikit-learn solver`
+
+[Read more in the documentation](https://intelpython.github.io/daal4py/sklearn.html#scikit-learn-verbose).
+
+# Building from Source
+See [Building from Sources](INSTALL.md) for details.
```

Comment:
I think this should be a subsection in Installation (see my other comments in a previous review).